src/style.css

```diff
@@ -1,3 +1,87 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 50px;
+  margin-right: 50px;
+  width: 100%;
```

Comment:
Don't fix container size (if there is no such a requirement). Let the content size dictate it. To avoid overflow or accidental scroll bar.